tools/ConfigureGeneratedApplications/ConfigurationJsonSerializerContext.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Text.Json.Serialization;
+using System.Text.Json;
+
+using ConfigureGeneratedApplications.Model;
+
+namespace ConfigureGeneratedApplications
+{
+    [JsonSourceGenerationOptions(
+    ReadCommentHandling = JsonCommentHandling.Skip,
+    WriteIndented = true)]
```

Comment:
```suggestion
    [JsonSourceGenerationOptions(ReadCommentHandling = JsonCommentHandling.Skip)]
```

This is only used to deserialize, not serialize. No need to configure writing

Reply:
Okay, removed the write indented. Thank you!